mcsema/Arch/X86/Semantics/Bitcode/Instruction.h

```diff
@@ -2,4 +2,10 @@
 
 #include "mcsema/Arch/X86/Runtime/State.h"
 
+#if _WIN32
```

Comment:
`State.h` defines a union for LONGDOUBLE on Win32, we need to get at the succulent innards of the union to get to the actual double value.